study-builder/studies/mbc/emails/followup_reminder_es.html

```diff
@@ -0,0 +1,221 @@
+            padding: 20px 0 20px 0 !important;
+          }
+
+          td[class="logo"] img{
+            margin:0 auto!important;
+          }
+
+          /* USE THESE CLASSES TO HIDE CONTENT ON MOBILE */
+          td[class="mobile-hide"]{
+            display:none;}
+
+          img[class="mobile-hide"]{
+            display: none !important;
+          }
+
+          img[class="img-max"]{
+            max-width: 100% !important;
+            height:auto !important;
+          }
+
+          /* FULL-WIDTH TABLES */
+          table[class="responsive-table"]{
+            width:100%!important;
+          }
+
+          /* UTILITY CLASSES FOR ADJUSTING PADDING ON MOBILE */
+          td[class="padding"]{
+            padding: 10px 5% 15px 5% !important;
+          }
+
+          td[class="padding-copy"]{
+            padding: 10px 5% 10px 5% !important;
+            text-align: left !important;
+          }
+
+          td[class="padding-meta"]{
+            padding: 30px 5% 0px 5% !important;
+            text-align: center;
+          }
+
+          td[class="no-pad"]{
+            padding: 0 0 20px 0 !important;
+          }
+
+          td[class="no-padding"]{
+            padding: 0 !important;
+          }
+
+          td[class="section-padding"]{
+            padding: 10px 15px 10px 15px !important;
+          }
+
+          td[class="section-padding-bottom-image"]{
+            padding: 50px 15px 0 15px !important;
+          }
+
+          /* ADJUST BUTTONS ON MOBILE */
+          td[class="mobile-wrapper"]{
+              padding: 10px 5% 15px 5% !important;
+          }
+
+          table[class="mobile-button-container"]{
+              margin:0 auto;
+              width:100% !important;
+          }
+
+          a[class="mobile-button"]{
+              width:90% !important;
+              padding: 15px !important;
+              border: 0 !important;
+              font-size: 16px !important;
+          }
+
+      }
+    </style>
+  </head>
+  <body style="margin: 0; padding: 0;">
+    <!-- HEADER -->
+    <table border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td align="center" bgcolor="#ffffff">
+          <!-- HIDDEN PREHEADER TEXT -->
+          <div style="display: none; font-size: 1px; color: #fefefe; line-height: 1px; font-family: Helvetica, Arial, sans-serif; max-height: 0px; max-width: 0px; opacity: 0; overflow: hidden;">
+            Recordatorio del MBCproject: Cuestionario de Seguimiento #1
+          </div>
+          <table border="0" cellpadding="0" cellspacing="0" class="wrapper" width="500">
+            <!-- LOGO/PREHEADER TEXT -->
+            <tr>
+              <td align="center" class="logo" style="padding: 20px 0px 30px 0px;">
+                <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                  <tr>
+                    <td align="center" bgcolor="#ffffff" width="100">
+                      <a target="blank" href="-ddp.baseWebUrl-">
```

Comment:
Another! This one is hard to detect because it doesn't actually show up in the email text but is rather a link. Try doing a grep for `-ddp.baseWebUrl-`.